Terraform/GCP/k8s/values/elastic.yaml

```diff
@@ -343,9 +376,9 @@ metricbeat:
   deployment:
     extraEnvs:
       - name: "ES_ENDPOINT"
-        value: "http://elasticsearch-master.default.svc:9200"
+        value: "https://elasticsearch-master:9200"
       - name: "KIBANA_URL"
-        value: "http://elastic-kibana.default.svc:5601"
+        value: "https://elastic-kibana.:5601"
```

Comment:
probably should be just `elastic-kibana:5601`.

Also does etting Kibana UI to `https` work ok with Ingress for Kibana?